test/e2e/clusterclass_changes_test.go

```diff
@@ -54,7 +54,9 @@ var _ = Describe("When testing ClusterClass changes [ClusterClass]", func() {
 						"hostPath":      "/var/run/docker.sock",
 					},
 					map[string]interface{}{
-						"containerPath": "/tmp",
+						// /tmp cannot be used as containerPath as
+						// it already exists.
+						"containerPath": "/test",
```

Comment:
Because we only care about the fields being be propagated to the MD and not actually if the machines come up after that change (also makes the test faster)